src/game/BattleGround/BattleGround.h

```diff
@@ -616,6 +619,8 @@ class BattleGround
         float m_TeamStartLocZ[PVP_TEAM_COUNT];
         float m_TeamStartLocO[PVP_TEAM_COUNT];
         float m_startMaxDist;
+
+        GuidVector m_objUpdates;
```

Comment:
Why not use std::set instead. You require uniqueness, so that solves it for you implicitly.

Reply:
Yeah good point will update that